### PR TITLE
Search results2

### PR DESCRIPTION
**Pull request by VITALIKXXX**

### CODE REVIEW

File: src/common/Header/SearchBar/index.js
Comment (bartosz-kasprzyk) [inline src/common/Header/SearchBar/index.js:1]: On "working-searchbar" branch there already is a SearchBar component with necessary hooks

File: src/common/searchResults/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            } catch (e) {
+                setError("Unable to fetch data.");
+            } finally {
+                setLoading(false);
+            }
+        };
+
+        fetchData();
+
+    }, [query]);
```
Comment (bartosz-kasprzyk) [inline src/common/searchResults/index.js:35]: Downloading API data with debounce is in separate file useSearchResults.js

File: src/common/searchResults/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    if (movies.length === 0) {
+        return <NoResults query={query} />;
+    };
+
+    return (
+        <Container>
+            {movies.map(movie => (
+                <MovieTile key={movie.id} {...movie} />
+            ))}
+        </Container>
```
Comment (bartosz-kasprzyk) [inline src/common/searchResults/index.js:54]: There should be two different outcomes depending on what the person was searching for (movies/people)

**Review by bartosz-kasprzyk — CHANGES_REQUESTED** [review]

**Review by przemyslaw-pacek — APPROVED** [review]

File: src/common/Header/SearchBar/index.js
Comment (przemyslaw-pacek) [reply src/common/Header/SearchBar/index.js:1]: Maybe because "Search Results 2" was created before "working search bar".

**Pull request closed without merging**